src/Elastic.Apm.AspNetCore/AspNetCoreLogger.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using Elastic.Apm.AspNetCore.Extensions;
+using Elastic.Apm.Logging;
+using Microsoft.Extensions.Logging;
+using LogLevel = Elastic.Apm.Logging.LogLevel;
+
+namespace Elastic.Apm.AspNetCore
+{
+	internal class AspNetCoreLogger : IApmLogger
+	{
+		private readonly ILogger _logger;
+
+		public AspNetCoreLogger(ILoggerFactory loggerFactory)
+		{
+			_logger = loggerFactory?.CreateLogger("Apm") ?? throw new ArgumentNullException(nameof(loggerFactory));

```

Comment:
Could we maybe make it `ElasticApm` instead of `Apm`? That's be the same as the category we use in `appsettings.json`, and also we'd not take up the word that is used industry wide for ourself.

Reply:
changed